features/user/managing_customers/browsing_customers.feature

```diff
@@ -0,0 +1,17 @@
+@managing_customers
+Feature: Browsing customers
+    In order to see all customers in the store
+    As an Administrator
+    I want to browse customers
+
+    Background:
+        Given the store has customer "f.baggins@example.com"
+        And the store has customer "mr.banana@example.com"
+        And the store has customer "l.skywalker@example.com"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Browsing currencies in store
+        When I want to see all customers in store
+        Then I should see 4 customers in the list
```

Comment:
The issue is that there is no way to define who is admin and who is not. Making an assumption here about e-mail sounds bad. Anyway, I would say let's keep it this way - this is how Sylius is working currently. We will look at separating the staff of the store from customers.